ci_support/run_docker_build.sh

```diff
@@ -43,6 +43,16 @@ conda update --yes --all
 conda install --yes conda-build==1.18.2
 conda info
 
+# Install CUDA support in the docker container.
+export CUDA_RPM_VERSION="7.5-18"
+export CUDA_RPM_MD5="01fd3415b197f9ddc4e58f5a4f2a9af6"
+curl "http://developer.download.nvidia.com/compute/cuda/repos/rhel6/x86_64/cuda-repo-rhel6-${CUDA_RPM_VERSION}.x86_64.rpm" > cuda.rpm
+md5sum cuda.rpm | grep "${CUDA_RPM_MD5}"
+yum install -y cuda.rpm
```

Comment:
That is an open question for sure. There are three options that come to mind. Some could be mixed. There could also be other variants.

1. Package it and make it a dependency.
2. Create a pseudo package that merely verifies the system version is present and works.
3. Use the system version and document the requirement.

A related point is that GPU support probably should be an optional feature so that users can install this and get basic CPU support without GPU. However, users wanting GPU support (and possibly CUDA in particular) could have a feature package they install to enable this support.